src/PrestaShopBundle/Twig/Component/Toolbar.php

```diff
@@ -69,4 +71,22 @@ public function mount(): void
             $this->hookDispatcher->dispatchWithParameters('actionAdminBreadcrumbModifier', ['tabs' => $tabs, 'breadcrumb' => &$this->breadcrumbs]);
         }
     }
+
+    public function getToolbarBtn()
+    {
+        // todo: Find a better way when we definitively drop smarty...
+        return $this->context->getLegacyLayoutData()->toolbarBtn;
+    }
+
+    public function getEnableSidebar()
+    {
+        // todo: Find a better way when we definitively drop smarty...
+        return $this->context->getLegacyLayoutData()->enableSidebar;
+    }
+
+    public function getHelpLink()
+    {
+        // todo: Find a better way when we definitively drop smarty...
+        return $this->context->getLegacyLayoutData()->helpLink;
+    }
```

Comment:
I think we can remove the class variables `$toolbarBtn` `$enableSidebar` `$helpLink`